components/profileHandler.ts

```diff
@@ -537,11 +537,13 @@ profileRouter.post(
         }
 
         for (const challenge of challenges) {
+            // TODO: Add actual support for shortcut challenges
```

Comment:
Care to elaborate?

Reply:
Pretty sure this is my to-do from a while back

Reply:
Yeah it's #16

Reply:
Ah okay.